### PR TITLE
changelog for v2.12.5

### PR DESCRIPTION
**Pull request by kdomanski**
```release-note
NONE
```

/assign @xrstf

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+### [v2.12.5]()
+
+
+- A bug that occasionally resulted in a `Error: no matches for kind &#34;MachineDeployment&#34; in version &#34;cluster.k8s.io/v1alpha1&#34; visible in the UI was fixed. [#4870](https://github.com/kubermatic/kubermatic/issues/4870) ([alvaroaleman](https://github.com/alvaroaleman))
```
Comment (xrstf) [inline CHANGELOG.md]: The backtick quoting didn't really work here. The closing ` is missing.

Also why the HTML encoding?

**Review by xrstf — CHANGES_REQUESTED** [review]
Nits

File: CHANGELOG.md (hunk 1 above)
Comment (kdomanski) [reply CHANGELOG.md]: I fixed the quotation. The HTML encodings seems to be what we get from the API when `gchl` scrapes the release-notes from the github API. Usually it works just fine, except for inside backtick quotations.

**Pull request merged**